src/main.rs

```diff
@@ -348,6 +348,18 @@ fn main() -> Result<()> {
                 .long("danger-accept-invalid-certs")
                 .help("Controls the use of certificate validation."),
         )
+        .arg(
+            Arg::with_name("RESPONSE_OUTPUT_FORMAT")
+                .long("response-output-format")
```

Comment:
Let's make this one `-s`
```suggestion
                .long("response-output-format")
                .short("s")
```